### ISSUE TITLE
Quill failing to handle certain keyboard bindings

### ISSUE THREAD

**Issue by roobscoob** [quilljs/quill#3481]
Please describe the a concise description and fill out the details below. It will help others efficiently understand your request and get to an answer instead of repeated back and forth. Providing a [minimal, complete and verifiable example](https://stackoverflow.com/help/mcve) will further increase your chances that someone can help.

**Steps for Reproduction**

1. Create a quill keyboard binding:
```js
{
  key: 'b',
  ctrlKey: true, // or shortKey
}
```
2. Try to use the keyboard binding

**Expected behavior**:

our handler is executed

**Actual behavior**:

our handler is not executed

**Platforms**:

Windows 10, on Chrome 95

**Version**:

1.3.7

**Comment by roobscoob** [comment]
diving into this further, It's due to the early return if defaultPrevented
```js
  listen() {
    this.quill.root.addEventListener('keydown', (evt) => {
      if (evt.defaultPrevented) return;
      let which = evt.which || evt.keyCode;
      let bindings = (this.bindings[which] || []).filter(function(binding) {
```

**Comment by IamMathankumar** [comment]
This issue not resolved in 2.0.2 as well
